pandora-server-directory/src/spaces/space.ts

```diff
@@ -414,7 +410,13 @@ export class Space {
 			const invite = this._getValidInvite(character, inviteId);
 			if (!invite)
 				return 'invalidInvite';
-
+			if (invite.type === 'joinMe') {
+				const creator = [...this.characters].find((c) => c.baseInfo.id === invite.createdBy.characterId);
+				if (creator?.assignedClient == null)
+					return 'invalidInvite';
```

Comment:
Use `isOnline()` to check this instead; it is more consistent - not prone to rapid changes on temporary DCs.

Reply:
I copied the logic from `hasAdminInside()`, I updating both to use `isOnline()`